object-construction-checker/src/main/java/org/checkerframework/checker/objectconstruction/ObjectConstructionChecker.java

```diff
@@ -106,8 +107,9 @@ public void reportError(Object source, @CompilerMessageKey String messageKey, Ob
   @Override
   public void typeProcessingOver() {
     if (hasOption(COUNT_MUST_CALL)) {
-      System.out.printf("Found %d must call obligation(s).%n", numMustCall);
-      System.out.printf(
+      message(NOTE, "Found %d must call obligation(s).%n", numMustCall);
+      message(
+          NOTE,
           "Found %d must call obligation(s) that were handled correctly.%n",
```

Comment:
While we're here, can we clarify this message?
```suggestion
          "Successfully verified %d must call obligation(s).%n",
```